rllib/policy/sample_batch.py

```diff
@@ -24,49 +24,61 @@ class SampleBatch:
     samples, each with an "obs" and "reward" attribute.
     """
 
-    # Outputs from interacting with the environment
-    CUR_OBS = "obs"
-    NEXT_OBS = "new_obs"
+    # Outputs from interacting with the environment.
+    OBS = "obs"
     ACTIONS = "actions"
     REWARDS = "rewards"
-    PREV_ACTIONS = "prev_actions"
-    PREV_REWARDS = "prev_rewards"
     DONES = "dones"
     INFOS = "infos"
 
+    # Deprecated: Use OBS instead.
+    CUR_OBS = "obs"
+    # Deprecated: Use ViewRequirement for your Model instead with OBS (t+1).
+    NEXT_OBS = "new_obs"
+    # deprecated: Use ViewRequirement for your Model instead with ACTIONS (t-1)
+    PREV_ACTIONS = "prev_actions"
+    PREV_REWARDS = "prev_rewards"
+
     # Extra action fetches keys.
     ACTION_DIST_INPUTS = "action_dist_inputs"
     ACTION_PROB = "action_prob"
     ACTION_LOGP = "action_logp"
 
-    # Uniquely identifies an episode
+    # Uniquely identifies an episode.
     EPS_ID = "eps_id"
 
     # Uniquely identifies a sample batch. This is important to distinguish RNN
     # sequences from the same episode when multiple sample batches are
     # concatenated (fusing sequences across batches can be unsafe).
     UNROLL_ID = "unroll_id"
 
-    # Uniquely identifies an agent within an episode
+    # Uniquely identifies an agent within an episode.
     AGENT_INDEX = "agent_index"
 
-    # Value function predictions emitted by the behaviour policy
+    # Value function predictions emitted by the behaviour policy.
     VF_PREDS = "vf_preds"
 
     @PublicAPI
     def __init__(self, *args, **kwargs):
         """Constructs a sample batch (same params as dict constructor)."""
 
+        # Placeholder for a last_obs value (in an n x sars'-trajectory).
```

Comment:
This prop will be needed once we retire NEXT_OBS. It holds a single(!) observation at the very end of the trajectory (instead of all NEXT_OBS over all timesteps in the to-be-deprecated case). Some postprocessing functions will have to be changed to use this prop instead of NEXT_OBS (see e.g. PPO).